roles/frontend/templates/nginx-conf.d-flatpak-ssl.conf.j2

```diff
@@ -0,0 +1,54 @@
+server {
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+
+    server_name {{ frontend_nginx_flatpak_domain }} www.{{ frontend_nginx_flatpak_domain }};
+
+    location /.well-known/acme-challenge {
+      proxy_pass {{ frontend_nginx_flatpak_origin }}/.well-known/acme-challenge;
+      proxy_hide_header Strict-Transport-Security;
+      proxy_hide_header Vary;
+    }
+
+    location / {
+      proxy_cache frontend_cache;
+      proxy_hide_header Strict-Transport-Security;
+      proxy_hide_header Vary;
+      proxy_set_header Host $host;
+      proxy_pass {{ frontend_nginx_flatpak_origin }};
+    }
+
+    # Maintain disabling of gzip for svgz files
+    location ~* \.svgz$ {
+      gzip off;
+      add_header Content-Encoding "gzip";
+    }
+
+    # certs sent to the client in SERVER HELLO are concatenated in ssl_certificate
+    ssl_certificate /var/lib/certsync/certs/{{ frontend_nginx_flatpak_domain }}/fullchain.pem;
+    ssl_certificate_key /var/lib/certsync/certs/{{ frontend_nginx_flatpak_domain }}/privkey.pem;
+    ssl_session_timeout 1d;
+    ssl_session_cache shared:SSL:50m;
+    ssl_session_tickets off;
+
+    # Diffie-Hellman parameter for DHE ciphersuites, recommended 2048 bits
+    ssl_dhparam /etc/nginx/ffdhe2048.pem;
+
+    # intermediate configuration. tweak to your needs.
+    ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+    ssl_ciphers 'ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA:ECDHE-ECDSA-DES-CBC3-SHA:ECDHE-RSA-DES-CBC3-SHA:EDH-RSA-DES-CBC3-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:DES-CBC3-SHA:!DSS';
+    ssl_prefer_server_ciphers on;
+
+    # HSTS (ngx_http_headers_module is required) (15768000 seconds = 6 months)
+    add_header Strict-Transport-Security max-age=15768000;
+
+    # OCSP Stapling ---
+    # fetch OCSP records from URL in ssl_certificate and cache them
+    ssl_stapling on;
+    ssl_stapling_verify on;
+
+    ## verify chain of trust of OCSP response using Root CA and Intermediate certs
+    ssl_trusted_certificate /var/lib/certsync/certs/{{ frontend_nginx_flatpak_domain }}/chain.pem;
+
+    resolver 93.93.128.2 93.93.130.2;
```

Comment:
There's a variable for these - frontend_nginx_resolver defined in the mythic group.

Reply:
Heh. It's hardcoded all over the place. I'll see about fixing that.

Reply:
WHAT...?! WHO WOULD EVEN DO THAT?!

:innocent: 

Thanks.